doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -36,6 +36,15 @@ B<context-string> as input to sign and verify operations (and for
 Ed25519ctx, the context-string must be nonempty).  For the Ed25519
 instance, a nonempty context-string is not permitted.
 
+These instances can be specified as signature parameters when using
+L<EVP_DigestSignInit(3)> and L<EVP_DigestVerifyInit(3)>, see
+L</ED25519 and ED448 Signature Parameters> below.
+
+These instances are also explicitly fetchable as algorithms using
```

Comment:
I would have expected Ed25519ph and Ed25519ctx (and similar Ed448 instances) to be listed independently in the NAME section of this pod file - as well as under the "Asymmetric Signature" heading in OSSL_PROVIDER-default and OSSL_PROVIDER-fips.

Reply:
Didn't do that for RSA, and was quite apprehensive at the idea of adding more names.  Can't we see it as a collection, i.e. ED25519 and ED448 may suffice, and the manual expands on all the variants?

Reply:
I'd still prefer them listed. But I won't insist.